app/components/Send/SendPanel/SendRecipientList/SendRecipientListItem/index.jsx

```diff
@@ -48,6 +59,26 @@ class SendRecipientListItem extends Component<Props> {
 
     const { name } = e.target
     let { value } = e.target
+    if (name === 'address') this.setState({ nnsError: '' })
+    if (name === 'address' && value.endsWith('.neo')) {
+      this.setState({ isNnsResolving: true })
+      if (this.debounceTimer) clearTimeout(this.debounceTimer)
+      this.debounceTimer = setTimeout(() => {
+        resolveNnsDomain(value)
+          .then(address => {
+            clearErrors(index, name)
+            this.setState({ isNnsResolving: false })
```

Comment:
same comment here with setState callback. would look like this this.setState({ isNnsResolving: false }, () => doSomething())

Reply:
Just to unset the loading animation.